ParcelDistributionCenter/ParcelDistributionCenter.Web/Controllers/PackagesController.cs

```diff
@@ -52,26 +56,28 @@ public ActionResult Index()
         //{
         //    return View();
         //}
-        //// GET: PackagesController/Edit/5
-        //public ActionResult Edit(int id)
-        //{
-        //    return View();
-        //}
+        // GET: PackagesController/Edit/5
+        public ActionResult Edit(int id)
+        {
+            var model = PackageHandler.FindPackageByNumber(id);
+            return View(model);
+        }
 
-        //// POST: PackagesController/Edit/5
-        //[HttpPost]
-        //[ValidateAntiForgeryToken]
-        //public ActionResult Edit(int id, IFormCollection collection)
-        //{
-        //    try
-        //    {
-        //        return RedirectToAction(nameof(Index));
-        //    }
-        //    catch
-        //    {
-        //        return View();
-        //    }
-        //}
+        // POST: PackagesController/Edit/5
+        [HttpPost]
+        [ValidateAntiForgeryToken]
+        public ActionResult Edit(int id, Package package)
```

Comment:
Pytanie czy on potrzbeuje tutaj tego parametru "int id". Jeśli nie to pewnie będzie można dla czystości go wywalić